### PR TITLE
Fixes #87

### LINKED ISSUE #87: traveltimetc change
traveltimetc - sheet flow equations changed from old version

changed to 
=(0.42/SQRT(H19))*(H17*H16/SQRT(H18/100))^0.8
from 
=(0.42/SQRT(H19))*(H17*H16/SQRT((MAX(0.0001,H18)/100))^0.8)

### PR DESCRIPTION
**Pull request by harper-wavra**
changed to
=(0.42/SQRT(H19))(H17H16/SQRT(H18/100))^0.8
from
=(0.42/SQRT(H19))(H17H16/SQRT((MAX(0.0001,H18)/100))^0.8)

### CODE REVIEW

**Review by amedenblik — CHANGES_REQUESTED** [review]
The `traveltimetc` endpoint is giving this 500 error: 
```
{
  "detail": "float division by zero"
}
```

Upon further inspection of the spreadsheet, it turns out that the value of the Bermuda grass Overland Slope is not actually 0.00 but 0.000001: 
image

That is what is preventing the spreadsheet from having a similar "divide by 0" error.

My suggestion is to change the equation to:

~`travel_time_sheet_flow += (0.42 / math.sqrt(P2_24_2)) * (mannings_n*corrected_length/math.sqrt((max(0.000001,surface["Overland Slope"])/100.0))**0.8) # minutes`~

Need to look into this more.

**Comment by harper-wavra** [comment]
- added Exception if user enters a value for Overland Slope that is less than or equal to 0
- changed traveltimetc example request to match new spreadsheet

File: CHANGELOG.md [hunk 1]
```diff
@@ -9,12 +9,14 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
 
 ### Added 
 
--  
+-  Exception if user enters a value for Overland Slope that is less than or eqaul to 0
```
Comment (amedenblik) [inline CHANGELOG.md]: Typo "eqaul"

**Review by amedenblik — APPROVED** [review]

File: CHANGELOG.md (hunk 1 above)
Comment (amedenblik) [inline CHANGELOG.md]:
```suggestion
-  Exception if user enters a value for Overland Slope that is less than or equal to 0
```

**Pull request merged**